packages/wrangler/src/index.tsx

```diff
@@ -1463,13 +1084,9 @@ function createCLIParser(argv: string[]) {
 		}
 	);
 
-	wrangler.command(
-		"pages",
-		"⚡️ Configure Cloudflare Pages",
-		async (pagesYargs) => {
-			await pages(pagesYargs.command(subHelp));
-		}
-	);
+	wrangler.command("pages", "⚡️ Configure Cloudflare Pages", (pagesYargs) => {
+		return pages(pagesYargs.command(subHelp));
```

Comment:
@GregBrimble I fixed this for y'all - doesn't need to be async

Reply:
Tah!